int32 carType
---
bool isAllowed